dashboard/config/scripts/levels/hoc2015_event_add_items.level

```diff
@@ -34,7 +34,8 @@
     "remove_items_when_actor_collides": false,
     "markerHeight": 50,
     "markerWidth": 50,
-
+    "delayCompletion": true,
```

Comment:
I can change this to specify the delay in milliseconds for each .level file, rather than just a boolean.